google/cloud/aiplatform/featurestore/featurestore.py

```diff
@@ -556,3 +561,461 @@ def create_entity_type(
+                        'my_entity_type_id_2': ['feature_id_2_1', 'feature_id_2_2'],
+                    }
+
+            feature_destination_fields (Dict[str, str]):
+                Optional. A user defined dictionary to map a feature's fully qualified resource name to
+                its destination field name. If the destination field name is not defined,
+                the feature ID will be used as its destination field name.
+
+                Example:
+                    feature_destination_fields = {
+                        'projects/123/locations/us-central1/featurestores/fs_id/entityTypes/et_id1/features/f_id11': 'foo',
+                        'projects/123/locations/us-central1/featurestores/fs_id/entityTypes/et_id2/features/f_id22': 'bar',
+                     }
+
+            read_instances (Union[str, List[str]]):
+                Optional. Read_instances can be either BigQuery URI to the input table,
+                or Google Cloud Storage URI(-s) to the
+                csv file(s). May contain wildcards. For more
+                information on wildcards, see
+                https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
+                Example:
+                    'bq://project.dataset.table_name'
+                    or
+                    ["gs://my_bucket/my_file_1.csv", "gs://my_bucket/my_file_2.csv"]
+
+                Each read instance consists of exactly one read timestamp
+                and one or more entity IDs identifying entities of the
+                corresponding EntityTypes whose Features are requested.
+
+                Each output instance contains Feature values of requested
+                entities concatenated together as of the read time.
+
+                An example read instance may be
+                ``foo_entity_id, bar_entity_id, 2020-01-01T10:00:00.123Z``.
+
+                An example output instance may be
+                ``foo_entity_id, bar_entity_id, 2020-01-01T10:00:00.123Z, foo_entity_feature1_value, bar_entity_feature2_value``.
+
+                Timestamp in each read instance must be millisecond-aligned.
+
+                The columns can be in any order.
+
+                Values in the timestamp column must use the RFC 3339 format,
+                e.g. ``2012-07-30T10:43:17.123Z``.
+
+            pass_through_fields (List[str]):
+                Optional. When not empty, the specified fields in the
+                read_instances source will be joined as-is in the output,
+                in addition to those fields from the Featurestore Entity.
+
+                For BigQuery source, the type of the pass-through values
+                will be automatically inferred. For CSV source, the
+                pass-through values will be passed as opaque bytes.
+
+        Returns:
+            Featurestore - The featurestore resource object batch read feature values from.
```

Comment:
```suggestion
            Featurestore: The featurestore resource object batch read feature values from.
```

This formats correctly on Google Cloud docs.